### ISSUE TITLE
template in Typescript interface TimelineTooltipOption should be optional

### ISSUE THREAD

**Issue by kodemi** [visjs/vis-timeline#153]
Now interface is:
```
interface TimelineTooltipOption {
  followMouse?: boolean;
  overflowMethod?: 'cap' | 'flip' | 'none';
  delay?: number;
  template: (item: TimelineItem, editedData?: TimelineItem) => string;
}
```
Documentation says that default value for _template_ is _none_.

**Comment by mojoaxel** [comment]
@kodemi Thanks For pointing this out. Feel free to provide a pull-request if you want!

**Comment by yotamberk** [comment]
Closed with #177
